package.json

```diff
@@ -1064,7 +1069,7 @@
         "@types/ps-tree": "^1.1.0",
         "@types/request": "2.0.7",
         "@types/semver": "^5.5.0",
-        "@types/vscode": "1.57.0",
+        "@types/vscode": "^1.63.0",
```

Comment:
The minimum version of VS Code Functions supports (as marked by `engines.vscode` in package.json) is `1.66.0`. Should we just update the types to match that?